app.py

```diff
@@ -30,6 +30,16 @@
 __version__ = "0.1.2"
 __title__ = "si-cloc"
 
+from thoth.analyzer import __version__ as __analyzer__version__
+from thoth.common import __version__ as __common__version__
+
+__service_version__ = (
```

Comment:
`__component_version__`?

Reply:
Updated in 9cbf1055558ba38bf4c15d4838059163041a5ff3